### DISCUSSION TITLE
Caclulator Version

### DISCUSSION THREAD

**Discussion by Ctl-F** [CE-Programming/toolchain#341] [Q&A] [answered]
As a test I built the hello world example and transferred it to my calculator only to get an Version Error when I run the program. I then realized that this toolkit is for the TI-84 Plus CE and I have the 84 Plus C Silver Edition. Is there a setting or modification that I can make for my calculator version or am I just out of luck?

**Comment by adriweb** [comment, marked as answer]
You are indeed out of luck for the 84+CSE, this toolchain is strictly for the CE.
You can program in assembly for your calc however.

**Reply by Ctl-F** [reply]
Alright, thank you.
Do you have any reference/source that's good for learning assembly for the cse?
